book/forms.rst

```diff
@@ -344,17 +344,6 @@ object.
 
 .. configuration-block::
 
-    .. code-block:: yaml
```

Comment:
The best practices now prefer annotations over any other config format. With that, the order changed to Annotation, Yaml, Xml and PHP. So :+1: to do it now